Library/Homebrew/language/java.rb

```diff
@@ -1,16 +1,15 @@
 module Language
   module Java
-    def self.java_home_cmd(version = nil)
-      version_flag = " --version #{version}" if version
-      "/usr/libexec/java_home#{version_flag}"
-    end
+    def self.java_home_cmd(_version = nil); end
 
-    def self.java_home_env(version = nil)
-      { JAVA_HOME: "$(#{java_home_cmd(version)})" }
+    def self.java_home_env(_version = nil)
+      {}
     end
 
-    def self.overridable_java_home_env(version = nil)
-      { JAVA_HOME: "${JAVA_HOME:-$(#{java_home_cmd(version)})}" }
+    def self.overridable_java_home_env(_version = nil)
```

Comment:
I don't think this is a correct implementation. See the uses of this in Homebrew/homebrew-core; they won't work after this. It would be good to use something here that makes sense there.